condicionales/comparatorNumber-2-5.js

```diff
@@ -0,0 +1,26 @@
+// Escriba un programa que pida tres números y que escriba si son los tres iguales, si hay dos iguales o si son los tres distintos.
+const prompt = require("prompt-sync")();
+
+const firstNumber = Number(prompt("Enter a first number: "));
+const secondNumber = Number(prompt("Enter a  second number: "));
+const thirdNumber = Number(prompt("Enter a  third number: "));
+
+if (
+	firstNumber === secondNumber &&
+	secondNumber === thirdNumber &&
+	thirdNumber === firstNumber
```

Comment:
this case is over, if first is equal to second and second is equal to third, then the third is going to be equal to first

Reply:
you didn't improve this observation :)